src/Appium.Net/Appium/Android/AndroidDriver.cs

```diff
@@ -191,11 +191,18 @@ public ConnectionType ConnectionType
         public void LongPressKeyCode(int keyCode, int metastate = -1) =>
             AppiumCommandExecutionHelper.LongPressKeyCode(this, keyCode, metastate);
 
+        #region Device Network
+
+        public void ToggleAirplaneMode() => AndroidCommandExecutionHelper.ToggleAirplaneMode(this);
```

Comment:
Please add a docstring. Also, this endpoint only works in Android below version 7

Reply:
Is, "Only supports below API 24 (Android 7)" good enough. Btw the xml document inherits from the interface

Reply:
yes, should be good.
Also, it probably, makes sense to mention that on emulator that works in any version. Toggling data transfer works ONLY on emulator.

Reply:
Are you sure about that? I just ran the ToggleAiplaneMode test on an Android 8.1 emulator (api 27) and receive the below
```
An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/Users/X/Library/Android/sdk/platform-tools/adb -P 5037 -s emulator-5554 shell am broadcast -a android.intent.action.AIRPLANE_MODE --ez state false' exited with code 255'; Stderr: 'Security exception: Permission Denial: not allowed to send broadcast android.intent.action.AIRPLANE_MODE from pid=29600, uid=2000
```

Initially I ran tests on an Android 9 physical device

Reply:
As for the toggling data, you mean that Toggling Airplane mode (back on) will turn on everything but mobile data on real devices?

Reply:
I've double checked the server code. You're right, on emulator the same command is used as for a real device, so it should also fail. Sorry, for the confusion. I've probably mixed it with toggleData.

The toggleData stuff enables/disables data transfer on emulators. There is no API to do this via command line on real devices though

Reply:
Ok. i've updated the docs for toggleData since I moved it to the interface